### ISSUE TITLE
questions about style transfer

### ISSUE THREAD

**Issue by suke27** [ycjing/Neural-Style-Transfer-Papers#13]
Hi, i have a question for style transfer. currently style transfer almost use BN/IN and gram matrix in train process, my question is that, is there any solution which use full convolution layer solution for style transfer?

**Comment by ycjing** [comment]
Hi @suke27 ,

I don't think so. Normalization layer is critical for the success of style transfer, as analyzed in our AAAI oral paper: https://arxiv.org/abs/1911.06953

**Comment by ycjing** [comment]
Hi @suke27,

Does it solve your problem?

**Comment by suke27** [comment]
no, i want to remove BN or IN from my layer. because i can not do clip forward in case of BN and IN
